R/imageutils.R

```diff
@@ -1,17 +1,25 @@
 startPNG <- function(filename, width, height, res, ...) {
+  args <- rlang::list2(filename=filename, width=width, height=height, res=res, ...)
+
   # If quartz is available, use png() (which will default to quartz).
   # Otherwise, if the Cairo package is installed, use CairoPNG().
   # Finally, if neither quartz nor Cairo, use png().
   if (capabilities("aqua")) {
     pngfun <- grDevices::png
+  } else if ((getOption('shiny.useragg') %OR% TRUE) &&
+             nchar(system.file(package = "ragg"))) {
```

Comment:
BTW since you have an rlang dep, you could use `rlang::is_installed()`.

Reply:
Just curious, what's the rationale for using `requireNamespace()` instead of `system.file()` (in `is_installed`)?

Reply:
I don't think it's a particularly strong motivation, but `requireNamespace()` also actually checks that package can be loaded.

Reply:
Ok, thanks, we'll probably do this in batch in another PR, so I'll keep as is for now for consistency sake